sdl2/test/pixels_test.py

```diff
@@ -9,9 +9,6 @@
 
 RGBA32 = [0xFF000000, 0x00FF0000, 0x0000FF00, 0x000000FF]
 RGBX32 = [0xFF000000, 0x00FF0000, 0x0000FF00, 0]
-if sys.byteorder == "big":
-    RGBA32.reverse()
-    RGBX32.reverse()
 
```

Comment:
I think this change is functionally correct, but it seems confusing to call these `RGBA32` and `RGBX32` when they're the masks for SDL's `SDL_PIXELFORMAT_RGBA8888` and `SDL_PIXELFORMAT_RGBX8888`, in which the pixel buffer is an array of native-endian 32-bit words where the bytes in descending order of significance are red (MSB), green, blue, and either alpha or padding as appropriate (LSB).

(Not to be confused with SDL's `SDL_PIXELFORMAT_RGBA32`, in which the pixel buffer is an array of 4-byte pixels in which each group of 4 bytes are red, green, blue, alpha in that order, regardless of whether the first or last would be considered to be the MSB in native endianness.)

I'd call them `RGBA8888` and `RGBX8888` or something similar.